modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm

```diff
@@ -5,9 +5,6 @@
 /obj/structure/closet/secure_closet/warden
 	icon = 'modular_skyrat/modules/sec_haul/icons/lockers/closet.dmi'
 
-/obj/structure/closet/secure_closet/security
```

Comment:
Security area is still blue colour coded. Having bright red lockers is an eye sore, and will make them look bad.

Reply:
I'm going to be talking with mappers about resolving the issues with the map coloring of Security not following the standard color-coding soon, as it's holding us back from getting the new and improved brig and permabrig for multiple maps at this rate. It's just a matter of whether I need to make the change myself or if I can get some assistance in doing so.

Reply:
We got changes coming to the basic color palette of bluesec after talking with Xyel and a spriter. I'll keep ya posted.